notebooks/tutorial_time.ipynb

```diff
@@ -272,6 +272,62 @@
     "larger_calendar.visualize(add_freq=True)"
```

Comment:
<p>This needs to be fixed. It runs well locally on my machine. I think you haven't updated/run this notebook after changing your code. It must be <code>from .bokeh_plots import bokeh_visualization</code> , which is correct in your code. Just run it again.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/s2spy/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Yeah this seems to have gone wrong. Thanks for noticing!